src/twisted/test/test_proto_helpers.py

```diff
@@ -0,0 +1,238 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+from twisted.trial import unittest
+
+
+
+class DeprecationTests(unittest.TestCase):
+    """
+    Deprecations in L{twisted.test.proto_helpers}.
+    """
+    def test_AccumlatingProtocol(self):
+        """
+        L{proto_helpers.AccumlatingProtocol} is deprecated and
+        generates a warning.
+        """
+        from twisted.test.proto_helpers import AccumulatingProtocol
+        AccumulatingProtocol  # shush
+
+        warnings = self.flushWarnings(
+            [self.test_AccumlatingProtocol])
+        self.assertEqual(DeprecationWarning, warnings[0]['category'])
+        self.assertEqual(
+            "twisted.test.proto_helpers.AccumulatingProtocol was "
+            "deprecated in Twisted 19.3.0: Please use "
+            "twisted.internet.testing.AccumulatingProtocol instead",
+            warnings[0]['message'])
+        self.assertEqual(1, len(warnings))
```

Comment:
Also, if you fix the duplications, you could put an `assertIs` in to check that the deprecated alias points to the new, non-deprecated public name.

Reply:
Done.